Nathanfishcircleci patch 2 eol launch agent (#8711)
circleci/circleci-docs: jekyll/_cci2/runner-installation-docker.adoc

:experimental:
:machine:

This page describes how to install CircleCI's machine runner with the Docker executor on server. If you are looking to set up self-hosted runners in a private Kubernetes cluster, please visit the <<container-runner#,Container runner>> page.

[#new-recommended-method-container-runner]

> == New supported method for Docker Installation (container runner)

Reviewer: ```suggestion
== Supported method for Docker installation (container runner)
```

The <<container-runner#,container runner>> is the recommended approach for running containerized jobs on self-hosted runners. Container runner offers the ability to seamlessly define, publish, and use custom Docker images during job execution, as well as the ability to easily manage dependencies or libraries through custom Docker images instead of enumerating dependencies as part of `steps` in the `.circleci/config.yml` file.

[#machine-approach-with-docker]
== Machine-based approach with Docker

> NOTICE: The instructions below are no longer supported for Cloud customers. 

Reviewer: ```suggestion
CAUTION: The instructions below are not supported for Cloud customers. 
```
